clean-leather.lic

```diff
@@ -26,13 +27,20 @@ class CleanLeather
     @bag = @settings.crafting_container
     @bag_items = @settings.crafting_items_in_container
     @belt = @settings.outfitting_belt
+    @engineering_room = @settings.engineering_room
```

Comment:
It's not necessary for this PR, but these two rooms don't need to be instance vars (`@...`), they can be local.